### PR TITLE
Re-enable ppc64le build

### PR DESCRIPTION
**Pull request by t20100**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

ppc64le build has been disabled in PR #35 (commit https://github.com/conda-forge/statsmodels-feedstock/pull/35/commits/b560977bb9cf609c246a14b56f8463a2642751de).
This PR aims at re-enabling it.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -3,17 +3,15 @@ bot:
   automerge: true
 build_platform:
   linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```
Comment (t20100) [inline conda-forge.yml:6]: Use cross-compilation

File: conda-forge.yml [hunk 2]
```diff
@@ -3,17 +3,15 @@ bot:
...
   branch_name: main
   tooling_branch_name: main
-linux_ppc64le:
-  enabled: false
 provider:
   linux_aarch64: default
-  linux_ppc64le: None
   win: azure
-test_on_native_only: true
+test: native_and_emulated
```
Comment (t20100) [inline conda-forge.yml:17]: > This has been deprecated in favor of the [test](https://conda-forge.org/docs/maintainer/conda_forge_yml.html#test) top-level field. It is now mapped to test: native_and_emulated.

https://conda-forge.org/docs/maintainer/conda_forge_yml.html#test-on-native-only

**Comment by t20100** [comment]
@conda-forge-admin, please rerender

**Comment by t20100** [comment]
CI fails for pypy with the following error when importing `statsmodels`:

<details><summary>Logs:</summary>

```
import: 'statsmodels'
Traceback (most recent call last):
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/test_tmp/run_test.py", line 2, in <module>
    import statsmodels
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/statsmodels/__init__.py", line 1, in <module>
    from statsmodels.compat.patsy import monkey_patch_cat_dtype
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/statsmodels/compat/__init__.py", line 1, in <module>
    from statsmodels.tools._testing import PytestTester
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/statsmodels/tools/__init__.py", line 1, in <module>
    from .tools import add_constant, categorical
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/statsmodels/tools/tools.py", line 5, in <module>
    import pandas as pd
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/pandas/__init__.py", line 22, in <module>
    from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/pandas/compat/__init__.py", line 25, in <module>
    from pandas.compat.numpy import (
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
    from pandas.util.version import Version
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/pandas/util/__init__.py", line 2, in <module>
    from pandas.util._decorators import (  # noqa:F401
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/pandas/util/_decorators.py", line 14, in <module>
    from pandas._libs.properties import cache_readonly
  File "/Users/runner/miniforge3/conda-bld/statsmodels_1695799519243/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/pypy3.9/site-packages/pandas/_libs/__init__.py", line 13, in <module>
    from pandas._libs.interval import Interval
TypeError: type 'cython_function_or_method' is not an acceptable base type
```

</details>

**Comment by bashtage** [comment]
pandas driven failure.

**Comment by bashtage** [comment]
The latest pandas build seems to be badly broken.  From another build on a different project:

```
Traceback (most recent call last):
  File "D:\bld\arch-py_1695814791478\test_tmp\run_test.py", line 2, in <module>
    import arch
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\arch\__init__.py", line 2, in <module>
    from .univariate.mean import arch_model
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\arch\univariate\__init__.py", line 5, in <module>
    from arch.univariate import recursions_python
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\arch\univariate\recursions_python.py", line 9, in <module>
    from arch.compat.numba import jit
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\arch\compat\numba.py", line 5, in <module>
    from arch.utility.exceptions import PerformanceWarning
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\arch\utility\__init__.py", line 5, in <module>
    from arch.utility.cov import cov_nw
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\arch\utility\cov.py", line 5, in <module>
    from arch.typing import Float64Array
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\arch\typing.py", line 8, in <module>
    from pandas import DataFrame, Series, Timestamp
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\pandas\__init__.py", line 22, in <module>
    from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\pandas\compat\__init__.py", line 25, in <module>
    from pandas.compat.numpy import (
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\pandas\compat\numpy\__init__.py", line 4, in <module>
    from pandas.util.version import Version
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\pandas\util\__init__.py", line 2, in <module>
    from pandas.util._decorators import (  # noqa:F401
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\pandas\util\_decorators.py", line 14, in <module>
    from pandas._libs.properties import cache_readonly
  File "D:\bld\arch-py_1695814791478\_test_env\lib\site-packages\pandas\_libs\__init__.py", line 13, in <module>
    from pandas._libs.interval import Interval
TypeError: type 'cython_function_or_method' is not an acceptable base type
```

**Comment by bashtage** [comment]
Seeign it accross Windows, Linux and OSX too.

**Comment by t20100** [comment]
@conda-forge-admin, please rerender

**Comment by t20100** [comment]
Rebased to fix conflicts and re-run CI since issue should have been fixed in pypy in the meantime (https://github.com/conda-forge/pypy3.6-feedstock/issues/109)

**Comment by xhochy** [comment]
@conda-forge/statsmodels Can you re-review/merge this?

**Comment by bashtage** [comment]
@conda-forge-admin, please rerender

**Comment by bashtage** [comment]
Thanks for the reminder @xhochy. Merged now.

**Pull request merged**